roles/confluent.kafka_broker/templates/listener.j2

```diff
@@ -39,3 +39,15 @@ listener.name.{{listener.value.name|lower}}.sasl.enabled.mechanisms=SCRAM-SHA-25
 listener.name.{{listener.value.name|lower}}.scram-sha-256.sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required \
    username="{{sasl_scram_users.admin.principal}}" password="{{sasl_scram_users.admin.password}}";
 {% endif %}
+{% if listener['value']['sasl_protocol'] | default(sasl_protocol) | normalize_sasl_protocol == 'OAUTHBEARER' %}
+listener.name.{{listener.value.name|lower}}.sasl.enabled.mechanisms=OAUTHBEARER
+listener.name.{{listener.value.name|lower}}.oauthbearer.sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
```

Comment:
Might be use variable for org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule like you did for listener.value.server_callback and listener.value.login_callback?

Reply:
We haven't been for all the other JAAS configs. So this was just in sticking with established convention. I'm also not aware of multiple JAAS Modules for any given SASL Mechanism.